tests/ctst/world/Zenko.ts

```diff
@@ -169,8 +170,8 @@ export default class Zenko extends World<ZenkoWorldParameters> {
      * @param {object} result - contains both stderr and stdout from the CLI command.
      * @returns {boolean} - if the result is a success or a failure
      */
-    checkResult(result: Utils.Command | Utils.Command[]): boolean {
-        const usedResult: Utils.Command[] = Array.isArray(result) ? result : [result];
+    checkResult(result: ApiResult | ApiResult[]): boolean {
+        const usedResult: ApiResult[] = Array.isArray(result) ? result : [result];
```

Comment:
Alternatively we could rename it `checkResults` and only accept arrays, so we can only call it with something like `checkResult([myResultObject])` if we only have one